website/src/components/symbols/HeaderImageRight.js

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+const HeaderImageRight = ({ height, width }) => {
+	return (
+		<svg aria-labelledby="title-energy-right" width={width} height={height} viewBox="0 0 268 228">
+			<title>{'Slack'}</title>
```

Comment:
That's definitely also not a slack svg right? :)
Let's call this like `title-energy-right` and fix the title and `aria-labeldby` id
```suggestion
```